li-apache-kafka-clients/src/main/java/com/linkedin/kafka/clients/consumer/LiKafkaConsumerConfig.java

```diff
@@ -168,6 +172,12 @@
                 atLeast(1),
                 Importance.MEDIUM,
                 MAX_POLL_RECORDS_DOC)
+        .define(METADATA_MAX_AGE_CONFIG,
+                Type.INT,
+                5 * 60 * 1000,
```

Comment:
is this available as a constant in kafka code somewhere ?

Reply:
Yes, in CommonClientConfigs, ProducerConfig, and ConsumerConfig. 

Actually, I didn't have to redefine them in LiKafkaConsumerConfig. As long as I define a default value here, it should be okay. Removed the definition and used CommonClientConfigs.METADATA_MAX_AGE_CONFIG/DOC here instead.